### PR TITLE
Dev asoundfix 2

### PR DESCRIPTION
**Pull request by Langerz82**
For reference to fixes see original PR:
https://github.com/EmuELEC/EmuELEC/pull/1138

### CODE REVIEW

**Comment by Langerz82** [comment]
@shantigilbert I have added fixes to the configuration of asound.

File: packages/sx05re/emuelec/bin/emuelec_autostart.sh [hunk 1]
```diff
@@ -16,20 +16,13 @@ if [ ! -L "$CONFIG_DIR" ]; then
...
-
-    if [ "$(get_es_setting bool StopMusicOnScreenSaver)" != "false" ]; then 
-        sed -i "/<bool name=\"StopMusicOnScreenSaver.*/d" "${ES_CONF}"
-        sed -i "s|</config>|	<bool name=\"StopMusicOnScreenSaver\" value=\"false\" />\n</config>|g" "${ES_CONF}"
-    fi
-
-elif [ "${EE_DEVICE}" == "Amlogic-ng" ]; then
-  rm /storage/.config/asound.conf > /dev/null 2>&1
-  cp /storage/.config/asound.conf-amlogic-ng /storage/.config/asound.conf
+if [ "$(get_es_setting bool StopMusicOnScreenSaver)" != "false" ]; then
```
Comment (shantigilbert) [inline packages/sx05re/emuelec/bin/emuelec_autostart.sh:19]: What does this have to do with this PR?

Comment (Langerz82) [reply packages/sx05re/emuelec/bin/emuelec_autostart.sh:19]: wait a sec, there is a better way to this, give me about half an hour.

Comment (Langerz82) [reply packages/sx05re/emuelec/bin/emuelec_autostart.sh]: Okay, you can check it now. The screensaver stuff is legacy code for amlogic devices, do we need it? Basically the default asound device file only gets copied to asound.conf if it doesn't exist. and the utils function just now works on asound.conf. 

Good for you?

Comment (Langerz82) [reply packages/sx05re/emuelec/bin/emuelec_autostart.sh]: one more thing, should we also in setauddev() set the retroarch config audio_device? Allot of people have no idea how to set that via menu.

**Comment by Langerz82** [comment]
Not really needed, since the copy will only occur once when starting up ES.
